jormungandr/src/network/p2p/policy.rs

```diff
@@ -40,6 +41,8 @@ pub struct PolicyConfig {
     max_quarantine: Option<Duration>,
     #[serde(default)]
     max_num_quarantine_records: Option<usize>,
+    #[serde(default)]
+    quarantine_whitelist: Option<HashSet<Address>>,
```

Comment:
the default of `HashSet` is already an empty `HashSet`:
```suggestion
    #[serde(default)]
    quarantine_whitelist: HashSet<Address>,
```

Reply:
Ok, it makes sense. Actually I had in mind that in some cases we would not want that whitelist, and my mind pop up `Option` instantly.